cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/TimeSeriesCategoryEndpointInput.java

```diff
@@ -125,7 +125,7 @@ protected HttpRequestBuilder addInputParameters(HttpRequestBuilder httpRequestBu
     public static final class Delete extends EndpointInput {
         private final String categoryId;
         private final String officeId;
-        private boolean cascadeDelete;
+        private boolean cascadeDelete = false;
```

Comment:
This is already the default behavior for boolean initialization. Fine to include it, but also fine to leave it off.